README.md

```diff
@@ -96,12 +96,51 @@ export PATH="${PATH}:${HOME}/.local/bin/"
 
 UtkarshVerma was so kind to provide an installation script for debian based systems, ![check it out here](https://github.com/UtkarshVerma/installer-scripts).
 
+#### Void Linux
+
+##### xbps repository
+
+`betterlockscreen` is available in official Void's repository as `betterlockscreen`.
+
+Installing using `xbps` (will automatically install all required dependencies): `xbps-install -S betterlockscreen`
+
 ### Configuration
 
 You can customise various colors for betterlockscreen, copy config file from examples directory to `~/.config/betterlockscreenrc` and edit it accordingly.
 
 If configuration file is not found then default configurations will be used.
```

Comment:
Could we hold it basic and link the betterlockscreenrc which we already provide? I also thought about changing the script to parse this included (not to modify - more a template) config to have a single source of truth for our defaults?

Reply:
> Could we hold it basic and link the betterlockscreenrc which we already provide?
Yes. I guess there's really no need for it in the README.

> I also thought about changing the script to parse this included (not to modify - more a template) config to have a single source of truth for our defaults?

Sounds like #191. We can make a config in `/etc/` or `~/.config/` or where ever, but the script will always need to certain amount of defaults included to be able to run without a config file.